tests/modules/blockonbackground.c

```diff
@@ -108,6 +134,10 @@ int HelloBlock_RedisCommand(RedisModuleCtx *ctx, RedisModuleString **argv, int a
     pthread_t tid;
     RedisModuleBlockedClient *bc = RedisModule_BlockClient(ctx,HelloBlock_Reply,HelloBlock_Timeout,HelloBlock_FreeData,timeout);
 
+    BlockPrivdata *block_privdata = RedisModule_Calloc(1, sizeof(*block_privdata));
+    block_privdata->measuretime_mutex = (pthread_mutex_t)PTHREAD_MUTEX_INITIALIZER;
+    RedisModule_BlockClientSetPrivateData(bc, block_privdata);
```

Comment:
this is repeated 3 times, let's add a method

Reply:
done with [`42de713` (#12817)](https://github.com/redis/redis/pull/12817/commits/42de713eb6cbd250b24903ab0caac93cdc86c41b)